apps/crn-frontend/src/shared-research/state.ts

```diff
@@ -156,17 +154,13 @@ const setResearchOutput = selector<ResearchOutputResponse | undefined>({
   },
 });
 
-export const usePostTeamResearchOutput = () => {
-  const authorization = useRecoilValue(authorizationState);
+export const useRefreshResearchOutputListing = () => {
```

Comment:
I think if we increase the number from the other place we can remove this hook as it should never be called in isolation

Reply:
Done